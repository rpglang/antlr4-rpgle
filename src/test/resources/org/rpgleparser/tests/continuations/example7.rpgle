     O                                           80 'Mary had a little lamb, its-
       
      *
      * Only a comment or a completely blank line is allowed in here
     O                                              fleece was white as snow.'