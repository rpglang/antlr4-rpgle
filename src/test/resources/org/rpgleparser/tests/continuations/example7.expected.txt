/*===TOKENS===*/
OS_FIXED                 <O>
OS_FieldReserved         <              >
BlankFlag                < >
BlankIndicator           <  >
BlankFlag                < >
BlankIndicator           <  >
BlankFlag                < >
BlankIndicator           <  >
OS_FieldName             <              >
OS_EditNames             < >
OS_BlankAfter            < >
OS_EndPosition           <   80>
OS_DataFormat            < >
StringLiteralStart       <'>
StringContent            <Mary had a little lamb, its>
StringContent            <fleece was white as snow.>
StringLiteralEnd         <'>
/*===TREE===*/
(r
  (statement
    (ospec_fixed
      O
      (os_fixed_pgmfield
        '              '
        (outputConditioningOnOffIndicator
          (onOffIndicatorsFlag ' ')
          (outputConditioningIndicator '  ')
        )
        (outputConditioningOnOffIndicator
          (onOffIndicatorsFlag ' ')
          (outputConditioningIndicator '  ')
        )
        (outputConditioningOnOffIndicator
          (onOffIndicatorsFlag ' ')
          (outputConditioningIndicator '  ')
        )
        '              '
        ' '
        ' '
        '   80'
        ' '
        (literal ' Mary had a little lamb, its fleece was white as snow. ')
      )
      <EOF>
    )
  )
)
/*======*/